src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultEditorTemplates.cs

```diff
@@ -374,6 +374,22 @@ public static string NumberInputTemplate(IHtmlHelper htmlHelper)
             return GenerateTextBox(htmlHelper, inputType: "number");
         }
 
+        public static string FileInputTemplate(IHtmlHelper htmlHelper)
```

Comment:
Not sure why none of the methods in this class have `[NotNull]` but we should probably add them for each of the methods you added.